### ISSUE TITLE
Multiple Issues w/ the Security Station

### ISSUE THREAD

**Issue by HipHopHuman** [TeamPneumatic/pnc-repressurized#672]
### Minecraft Version
1.16.4

### Forge Version
35.1.7

### Mod Version
1.16.4-2.7.2-87

### Describe your problem, including steps to reproduce it

There are multiple issues with the Security Station in 1.16.4, I'll do my best to illustrate them.

#### Issue 1: Broken Network Path Rendering

The "line" that is drawn between the different components in the Security Station, connecting them together, are no longer being drawn correctly. Instead, they draw from the top left corner, like this:

![image](https://user-images.githubusercontent.com/70091260/101267566-b6e95600-3762-11eb-9b4a-63a066909c80.png)

The line should go from the entry point to the other nodes. This is not unique to the particular pattern I am showing in this image, it happens consistently with any pattern I put in.

#### Issue 2: Unlocalized Names in "Test Security"

The info panels and tool tips in the "Test Security" interface are showing raw unlocalized names:

![image](https://user-images.githubusercontent.com/70091260/101267601-19daed00-3763-11eb-8c61-b9450a0445cf.png)

#### Issue 3: *Critically* Severe Lag Generation

There is a significant scaling lag effect in the security station that can potentially be used to crash a server. You can reproduce it by replicating the same pattern that I have in the above two images. Just wait a few minutes, and your FPS will drop to 1. TPS will drop considerably as well.

**Comment by desht** [comment]
Yep, the security station is quite broken, I know. I actually started a git branch to reimplement it from the ground up, but it had rather low priority, since I suspect it's the least used block in the mod. In fact, you're the first person to complain about it since the 1.14.4 port.

Guess I should dust off that branch and get it finished...

**Comment by HipHopHuman** [comment]
Oh I'm not complaining, just reporting bugs as I see them. I mostly play single player so the security station isn't quite something I'll use very often, but there's a certain part of the mod (Remotes) that requires you at least make a security station and place it down, so I did, and noticed the bugs.

**Comment by desht** [comment]
You don't *need* a security station to use the remote...

**Comment by HipHopHuman** [comment]
You don't? So it just makes it so that only you can edit the remote? I read the tooltip text which gave the impression that a Security Station was required: 
![image](https://user-images.githubusercontent.com/70091260/101279229-81c41e80-37c9-11eb-9159-d9c7da2719c2.png)

**Comment by desht** [comment]
That text is a bit misleading, admittedly.  Should probably read something like:
```
Tip: you can bind this Remote to a Security Station by right clicking it.
If you do, only players allowed by that Security Station can edit this Remote.
```
i.e. it's optional if you want to protect your remote.

**Comment by desht** [comment]
Build 89 fixes the GUI brokenness (bad rendering and client memory exhaustion), but note that security station isn't currently hackable.  However, it does a good job of protecting your base in SMP.

**Comment by HipHopHuman** [comment]
Nice, it performs a hell of a lot better now. Great work! One small suggestion: increase the thickness of the network "path" line. I am short-sighted, and can only see them with my super extra strong glasses (This is with 3 GUI Scale): 

![image](https://user-images.githubusercontent.com/70091260/101870305-0330fd80-3b8a-11eb-8470-922f2ce8052d.png)

**Comment by desht** [comment]
Fixed in 2.8.0 release
